MinimOsd_Extra/Panels.h

```diff
@@ -1286,9 +1286,12 @@ static void panGPSats(point p){
        gps_str = 0x20;
 
     if(has_sign(p))
-	OSD::write_S(gps_str);
+      osd.printf_P(PSTR("%2i%c"), osd_satellites_visible, gps_str);
+     else
+      osd.printf_P(PSTR("%2i"), osd_satellites_visible);
+	    //OSD::write_S(gps_str);
 
-    osd_printi_1(PSTR("%2i"), osd_satellites_visible);
+    //osd_printi_1(PSTR("%2i"), osd_satellites_visible);
```

Comment:
Moreover there is a set of cheap replacements for printf  - and you try  to return the code at the time when the previous developer said "all, nothing else will not go." Well, do not goes because printf everywhere!